pkg/nsx/services/vpc/vpc.go

```diff
@@ -974,3 +976,39 @@ func (s *VPCService) GetNSXLBSPath(lbsId string) string {
 	}
 	return *vpcLBS.Path
 }
+
+func GetAlbEndpoint(cluster *nsx.Cluster) error {
+	_, err := cluster.HttpGet(albEndpointPath)
+	return err
+}
+
+func (vpcService *VPCService) GetLbProvider() string {
```

Comment:
Do you want to change existing lb type check to this method? Check `NSXLBEnabled()` and `UseAVILoadBalancer` in whole project.

Reply:
it could be used to replace the logic inside NSXLBEnabled(), NSXLBEnabled could call GetLbProvider() == "nsx_lb"

Reply:
I agree, please replace NSXLBEnabled() or UseAVILoadBalancer()

Reply:
move NSXLBEnabled to vpc